radio/src/targets/horus/board.h

```diff
@@ -684,6 +684,21 @@ void bluetoothWriteWakeup();
 uint8_t bluetoothIsWriting();
 void bluetoothDisable();
 
+//OW
+// MAVLINK_TELEM driver
+//XX#if defined(MAVLINK_TELEM)
+#if defined(RADIO_T16) && defined(PCBX10)
```

Comment:
pleonasm?

Reply:
again, I followed pattern I have seen elsewhere

the flags in OpenTx are so complicated that I won't want to pretend competence here, so I just  follow pattern.

pl advice

BTW: Because of that I also have given up for the moment to figure out how to have everything enabled/disabled with a MAVLINK_TELEM flag. That's why you somewhere ind it but in other locations there it probably should be not. I would need your advice/insight here (but I think we can sort this out in the final step)

Reply:
@3djc ?

Reply:
The RADIO_XXX defines include all the underlying PCB / platform defines

Reply:
Looked briefly, looks like you use USART3, that is free only on X10 and T16, so I guess you meant #if defined(RADIO_T16) || (defined(PCBX10) && !defined(PCBREV)), or even better define a RADIO_X10 in horus cmake like we have for t16 and tx16s

Reply:
`#if defined(TELEMETRY_MAVLINK)` as explained in the other comment

Reply:
done. thx.

Reply:
@3djc yes, I'm using UART3 here. However, that's really just for the T16, and even for the T16 one IMHO can argue if it would be better to use the BLUETOOTH uart (which is unpopulated and not supported in code). Generally I think one eventually wants to have the UART configurable, e.g. for the TX16S (and you can see some precursors of that in the code). That is, I would think one would not want to make presumptions now which could be effortful to correct later.

Reply:
BT uart is uart6 and is populated/working in the code if BLUETOOTH option is enabled

Reply:
thx for the clarification. I was indeed a bit imprecise. I guess what I wanted to say is that - at least with v2.3.7 (haven't checked for v2.39) - bluetooth isn't normally enabled. Maybe this has changed, but my T16 hasn't had a bluetooth module and the pics of T16 I have seen also didn't had it populated. Maybe they've added it in the last few months however. The general point that one wants the uart to be configurable I think is however agreeable.